envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/EnvoyControlTestConfiguration.kt

```diff
@@ -37,11 +43,17 @@ object RandomConfigFile :
 abstract class EnvoyControlTestConfiguration : BaseEnvoyTest() {
```

Comment:
This class contains setup, http clients, stub applications interactions, SUT factories, SUT interactions, some assertions, and cleanup. I think we should consider splitting it :)

Reply:
I've created an issue for it. I'll work on it once mtls PR is one, ok? :) https://github.com/allegro/envoy-control/issues/113